_data/wcag-act-rules/act-implementations.yml

```
@@ -159,3 +159,10 @@
   standards:
     - WCAG 2.0 Level A, AA
   jsonReport: https://raw.githubusercontent.com/Section508Coordinators/act-tt/main/act-trusted-tester.json
+- name: Webmate Accessibility Workbench
+  vendor: Testfabrik Consulting + Solutions AG
+  uniqueKey: webmate-accessibility-workbench
+  type: automated tool
```

```suggestion
  type: Automated tool
```

For now, the implementation is not listed in the [Implementations](https://deploy-preview-332--wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/) page.

Note: A case sensitive attribute value is not ideal...